src/alchemlyb/visualisation/convergence.py

```diff
@@ -1,21 +1,20 @@
 import matplotlib.pyplot as plt
+import pandas as pd
 from matplotlib.font_manager import FontProperties as FP
 import numpy as np
 
-def plot_convergence(forward, forward_error, backward, backward_error,
-                     units='kT', ax=None):
+from ..postprocessors.units import get_unit_converter
+
+def plot_convergence(*data, units='kT', ax=None):
     """Plot the forward and backward convergence.
 
     Parameters
     ----------
-    forward : List
-        A list of free energy estimate from the first X% of data.
-    forward_error : List
-        A list of error from the first X% of data.
-    backward : List
-        A list of free energy estimate from the last X% of data.
-    backward_error : List
-        A list of error from the last X% of data.
+    data : Dataframe or 4 Lists
+        Output Dataframe from
+        :func:`~alchemlyb.postprocessors.convergence.forward_backward_convergence`.
+        Or given explicitly as `forward`, `forward_error`, `backward`,
```

Comment:
link to tutorial-like text is good but a function should be clear about its own inputs — what shape are these arrays, what do they contain, ... definitely re-use some the text that you are deleting